modules/setting-up-topology-manager.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * scaling_and_performance/using-topology-manager.adoc
+
+[id="seting_up_topology_manager_{context}"]
+= Setting up Topology Manager
+
+.Prequisites
+* Configure the CPU Manager policy to be `static`. Refer to Using CPU Manager in the Scalability and Performance section.
```

Comment:
@vpickard I changed this from a step in the procedure to a prerequisites. Unfortunately we can't add an real link here due to our documentation conventions.